tasks/Add Task To GCI

```diff
@@ -0,0 +1 @@
+Create a new Character In blender!!!!!!
```

Comment:
Pls rename the task to something suitable and add some more documentation as mentioned in other tasks.